test/bio_base64_test.c

```diff
@@ -205,25 +205,21 @@ static int test_bio_base64_run(test_case *t, int llen, int wscnt)
 
     if (raw == NULL && t->bytes > 0) {
         TEST_error("out of memory");
-        return -1;
+        goto err;
     }
 
     out_len = t->bytes + 1024;
     out = OPENSSL_malloc(out_len);
     if (out == NULL) {
-        OPENSSL_free(raw);
         TEST_error("out of memory");
-        return -1;
+        goto err;
     }
 
     elen = genb64(t->prefix, t->suffix, raw, t->bytes, t->trunc, t->encoded,
                   llen, wscnt, &encoded);
     if (elen < 0 || (bio = BIO_new(BIO_s_mem())) == NULL) {
```

Comment:
What about this instead:
```c
if (!TEST_int_lt(elen, 0) || !TEST_ptr(bio = BIO_new(BIO_s_mem())))
    goto err;
```
It will give a better error message and is shorter.